server/StudentList.ts

```diff
@@ -0,0 +1,32 @@
+import Student from "./Student.ts"
+
+class StudentList {
+
+    private students: Array<Student>;
+
+    constructor(){
+        this.students = [];
+    }
+
+    size(): number{
+        return this.students.length;
+    }
+
+    addStudent(nick: string, name: string, surname: string): void{
+        this.students.push(new Student(nick, name, surname));
+    }
+
+    getStudent(index: number): Student{
+        return this.students[index];
```

Comment:
ładne ogolnie, ale cyz nie lepsze bedzie mapa id: student? w sensie bedzie mozna szybciej szukac, a szukanie po indekie w tablicy jest srednie, bo co jesli ona sie zmieni jakos? czujeesz mnie?

Reply:
z tym szybkim szukaniem to bym tak nie przesadzał studentów aż tak dużo nie ma. Drugi argument jak najbardziej, jak wiemy z doświadczenia student zmienny jest raz go nie ma raz jest, a usuwanie w tablicy jest nie wygodne